docs/overview/threat-model.md

```diff
@@ -28,6 +28,8 @@ Other responsibilities include protecting the integrity of configuration files a
 
 The integrator is ultimately responsible for auditing usage of internal and external plugins as these run on the host system and have access to configuration and secrets. When installing plugins from sources like NPM, you should vet these in the same way that you would vet any other package installed from that source.
 
+The integrator is also responsible for due diligence in maintaining the resolved NPM dependencies of their Backstage project. This involves ensuring that `yarn.lock` receives updated versions of packages that have vulnerabilities, when those fixed versions are in range of what the Backstage packages request in their respective `package.json` files. This is commonly done by employing automated tooling such as [Dependabot](https://dependabot.com/), [Snyk](https://snyk.io/), and/or [Renovate](https://docs.renovatebot.com/) on your own repository. When fixed versions exist that are _not_ in range of what Backstage packages request, or when larger operations such as switching out an entire dependency for another one is required, maintainers collaborate with contributors to try to address those dependency declarations in the main project as soon as possible.
```

Comment:
We talk about due diligence of package selection above, I'd think this is more of an ongoing maintenance thing right? Perhaps just "responsible for maintaining"?

Reply:
yeah, cut out those words now